ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_isis.schema.yml

```diff
@@ -337,16 +337,30 @@ keys:
             type: int
             convert_types:
               - str
-            min: 1
-            max: 300
-            description: Maximum interval between two SPFs in seconds.
+            description: |-
+              Maximum interval between two SPFs in seconds or milliseconds.
+              Range in seconds: <1-300>
+              Range in milliseconds: <1-300000>
+          interval_unit:
+            type: str
+            valid_values:
+              - "seconds"
+              - "milliseconds"
+            default: seconds
```

Comment:
We may not need default for this case as if we do not defined interval_unit then we dont want any unit

Reply:
if interval unit is not defined then by default interval will be in seconds

Reply:
updated it!